draft-bormann-cbor-notable-tags.md

```diff
@@ -605,6 +605,9 @@ There are no special considerations for deterministic encoding
 overlap; particularly, tag 101 encoding starts where the more compact
 special encodings for 0..6 and 7..127 end.
 
+For cases 0..6 and 7..127, the tag value indicates the value of the alternatives.
+For cases 128+, then a single tag value announces the format, and the value within the array indicates the value.
```

Comment:
I think this can be said more understandably.

```suggestion
For cases 128+, a single tag number is used with an enclosed two-element array that contains the case number and the value of the alternative.
```